### ISSUE TITLE
Autosomal Dominant with maternal/paternal imprinting

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7485]
**Preferred term label:**
Autosomal Dominant with maternal/paternal imprinting

**Synonyms**

**Definition (free text, please give PubMed ID)**
A type of autosomal dominant inheritance involving a gene that is imprinted with either maternal or paternal silencing.

**Parent term (use hpo.jax.org/app)**
Autosomal dominant inheritance HP:0000006

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
The decision was to place the imprinted concept into the modifier section, so this issue is moot
